providers/core/core.oauth.js

```diff
@@ -98,8 +100,11 @@ OAuth.prototype.launchAuthFlow = function(authUrl, stateObj, continuation) {
     });
     return;
   }
-
-  this.ongoing[stateObj.state].launchAuthFlow(authUrl, stateObj, continuation);
+  if (interactive === undefined) {
+    interactive = true;
```

Comment:
What's the reasoning behind having interactive be the default? Will it be the most common use case?